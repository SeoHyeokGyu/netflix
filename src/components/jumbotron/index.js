import React from "react";

export default function Jumbotron(){
    return (
        <p>Jumba</p>
    );
}