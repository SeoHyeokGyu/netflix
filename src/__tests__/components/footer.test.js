import React from "react";
import {render, fireEvent} from "@testing-library/react";
import { Footer } from "../../components";

describe('<Footer />', () => {
    it('renders the <Footer /> with populated data', () => {
        const {container, getByText} = render( <Footer>
            <Footer.Title>
                질문이 있으신가요? 문의 전화: 00-000-000-0000
            </Footer.Title>
            <Footer.Break/>
            <Footer.Row>
                <Footer.Column>
                    <Footer.Link href="#">자주 묻는 질문</Footer.Link>
                    <Footer.Link href="#">투자 정보(IR)</Footer.Link>
                    <Footer.Link href="#">개인정보</Footer.Link>
                    <Footer.Link href="#">속도 테스트 </Footer.Link>
                </Footer.Column>

                <Footer.Column>
                    <Footer.Link href="#">고객 센터</Footer.Link>
                    <Footer.Link href="#">입사 정보</Footer.Link>
                    <Footer.Link href="#">쿠키 설정</Footer.Link>
                    <Footer.Link href="#">법적 고지</Footer.Link>
                </Footer.Column>

                <Footer.Column>
                    <Footer.Link href="#">계정</Footer.Link>
                    <Footer.Link href="#">Netflix 지원 디바이스</Footer.Link>
                    <Footer.Link href="#">회사 정보</Footer.Link>
                    <Footer.Link href="#">Netflix 오리지널</Footer.Link>
                </Footer.Column>

                <Footer.Column>
                    <Footer.Link href="#">미디어 센터</Footer.Link>
                    <Footer.Link href="#">이용 약관</Footer.Link>
                    <Footer.Link href="#">문의하기</Footer.Link>
                </Footer.Column>

            </Footer.Row>
            <Footer.Break/>
            <Footer.Text>Netflix 대한민국</Footer.Text>
            <Footer.Text>넷플릭스서비시스코리아 유한회사 통신판매업신고번호: 제2018-서울종로-0426호 전화번호: 00-308-321-0058
                <br />
                대표: 레지널드 숀 톰프슨
                <br />
                이메일 주소: korea@netflix.com
                <br />
                주소: 대한민국 서울특별시 종로구 우정국로 26, 센트로폴리스 A동 20층 우편번호 03161
                <br />
                클라우드 호스팅: Amazon Web Services Inc.
                <br />
                공정거래위원회 웹사이트 링크
            </Footer.Text>
        </Footer>
        );

        expect(getByText('질문이 있으신가요? 문의 전화: 00-000-000-0000')).toBeTruthy();
        expect(getByText('자주 묻는 질문')).toBeTruthy();
        expect(getByText('투자 정보(IR)')).toBeTruthy();
        expect(getByText('Netflix 지원 디바이스')).toBeTruthy();
        expect(getByText('회사 정보')).toBeTruthy();
        expect(getByText('Netflix 오리지널')).toBeTruthy();
        expect(container.firstChild).toMatchSnapshot();



    });
})