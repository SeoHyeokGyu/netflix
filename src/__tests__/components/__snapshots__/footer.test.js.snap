// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders the <Footer /> with populated data 1`] = `
<div
  class="footer__Container-qu6i6-0 ktDBMK"
>
  <p
    class="footer__Title-qu6i6-4 htVotQ"
  >
    질문이 있으신가요? 문의 전화: 00-000-000-0000
  </p>
  <p
    class="footer__Break-qu6i6-6 lblvIr"
  />
  <div
    class="footer__Row-qu6i6-2 kYqDwo"
  >
    <div
      class="footer__Column-qu6i6-1 canvup"
    >
      <a
        class="footer__Link-qu6i6-3 krWIOC"
        href="#"
      >
        자주 묻는 질문
      </a>
      <a
        class="footer__Link-qu6i6-3 krWIOC"
        href="#"
      >
        투자 정보(IR)
      </a>
      <a
        class="footer__Link-qu6i6-3 krWIOC"
        href="#"
      >
        개인정보
      </a>
      <a
        class="footer__Link-qu6i6-3 krWIOC"
        href="#"
      >
        속도 테스트 
      </a>
    </div>
    <div
      class="footer__Column-qu6i6-1 canvup"
    >
      <a
        class="footer__Link-qu6i6-3 krWIOC"
        href="#"
      >
        고객 센터
      </a>
      <a
        class="footer__Link-qu6i6-3 krWIOC"
        href="#"
      >
        입사 정보
      </a>
      <a
        class="footer__Link-qu6i6-3 krWIOC"
        href="#"
      >
        쿠키 설정
      </a>
      <a
        class="footer__Link-qu6i6-3 krWIOC"
        href="#"
      >
        법적 고지
      </a>
    </div>
    <div
      class="footer__Column-qu6i6-1 canvup"
    >
      <a
        class="footer__Link-qu6i6-3 krWIOC"
        href="#"
      >
        계정
      </a>
      <a
        class="footer__Link-qu6i6-3 krWIOC"
        href="#"
      >
        Netflix 지원 디바이스
      </a>
      <a
        class="footer__Link-qu6i6-3 krWIOC"
        href="#"
      >
        회사 정보
      </a>
      <a
        class="footer__Link-qu6i6-3 krWIOC"
        href="#"
      >
        Netflix 오리지널
      </a>
    </div>
    <div
      class="footer__Column-qu6i6-1 canvup"
    >
      <a
        class="footer__Link-qu6i6-3 krWIOC"
        href="#"
      >
        미디어 센터
      </a>
      <a
        class="footer__Link-qu6i6-3 krWIOC"
        href="#"
      >
        이용 약관
      </a>
      <a
        class="footer__Link-qu6i6-3 krWIOC"
        href="#"
      >
        문의하기
      </a>
    </div>
  </div>
  <p
    class="footer__Break-qu6i6-6 lblvIr"
  />
  <p
    class="footer__Text-qu6i6-5 cSwbkh"
  >
    Netflix 대한민국
  </p>
  <p
    class="footer__Text-qu6i6-5 cSwbkh"
  >
    넷플릭스서비시스코리아 유한회사 통신판매업신고번호: 제2018-서울종로-0426호 전화번호: 00-308-321-0058
    <br />
    대표: 레지널드 숀 톰프슨
    <br />
    이메일 주소: korea@netflix.com
    <br />
    주소: 대한민국 서울특별시 종로구 우정국로 26, 센트로폴리스 A동 20층 우편번호 03161
    <br />
    클라우드 호스팅: Amazon Web Services Inc.
    <br />
    공정거래위원회 웹사이트 링크
  </p>
</div>
`;
